recipes/build/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "build" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/build-{{ version }}.tar.gz
+  sha256: 5c6ca6cba06ea9a51623ce3197f019db8925ef3f06bf6d983fe5dcb16fcd1f6e
+
+build:
+  number: 0
+  skip: true  # [py<=34]
```

Comment:
`noarch: python`?

Reply:
`importlib-metadata` is `noarch` so it could avoid having the selector

Reply:
Yep. Thanks!